src/reports/timeline.rkt

```diff
@@ -361,20 +361,25 @@
   `((dt "Explanations")
     (dd (details
          (summary "Click to see full explanations table")
-         (table ((class "times"))
-                (thead (tr (th "Operator") (th "Subexpression") (th "Explanation") (th "Count")))
-                ,@(append*
-                   (for/list ([rec (in-list (sort explanations > #:key fourth))])
-                     (match-define (list op expr expl cnt mcnt flows) rec)
-
-                     (append (list `(tr (td (code ,(~a op)))
-                                        (td (code ,(~a expr)))
-                                        (td (b ,(~a expl)))
-                                        (td ,(~a cnt))
-                                        (td ,(~a mcnt))))
-                             (for/list ([flow (in-list (or flows '()))])
-                               (match-define (list ex type v) flow)
-                               `(tr (td "↳") (td (code ,(~a ex))) (td ,type) (td ,(~a v))))))))))))
+         (table
+          ((class "times"))
+          (thead
+           (tr (th "Operator") (th "Subexpression") (th "Explanation") (th "Count") (th "Locations")))
```

Comment:
I don't really understand why we need to print "locations" here

Reply:
Removed it.